components/input.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Icon link -->
+    <link rel="icon" href="../assets/icon.png">
+    <!-- css style links -->
+
+    <link rel="stylesheet" href="../css/component_styles/input_component.css">
+
+    <!-- title -->
+    <title>Image</title>
+
+    <!-- Font Awesome link -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+        integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V" crossorigin="anonymous" />
```

Comment:
you can include this in main css file so that you need not to repeatadly use it

Reply:
Yes, I will do it after 2 more component together